nsqlbproducer/lbproducer.go

```diff
@@ -0,0 +1,94 @@
+package nsqlbproducer
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/Scalingo/go-utils/nsqproducer"
+	nsq "github.com/nsqio/go-nsq"
+	errgo "gopkg.in/errgo.v1"
+)
+
+type NsqLBProducer struct {
+	producers []nsqproducer.Producer
```

Comment:
Can you add something to ensure `NsqLBProducer` implements all `nsqproducer.Producer` methods?